tests/acceptance/task-list-test.js

```diff
@@ -78,3 +79,41 @@ test('member can assign/reassign/unassign tasks to user', function(assert) {
     assert.ok(taskCard.taskAssignment.trigger.unassigned, 'Task is rendered unassigned.');
   });
 });
+
+test('clicking a task card is tracked', function(assert) {
+  assert.expect(1);
+
+  let project = server.create('project');
+  let { organization } = project;
+
+  let currentUser = createContributor(project);
+  let taskList = server.create('task-list', { project });
+  let task = server.create('task', { project, taskList });
+
+  authenticateSession(this.application, { user_id: currentUser.id });
+
+  page.visit({
+    organization: project.organization.slug,
+    project: project.slug
+  });
+
+  this.application.register('service:stubMetrics', Service.extend({
+    instantiate: false,
+    trackEvent(data) {
+      assert.deepEqual(data, {
+        event: 'Clicked on Task Card in List',
+        organization: organization.name,
+        organization_id: organization.id,
+        project: project.title,
+        project_id: project.id,
+        task: task.title,
+        task_id: task.id
+      }, 'Event tracking was called with proper attributes.');
+    }
+  }));
+  this.application.inject('route', 'metrics', 'service:stubMetrics');
```

Comment:
While I'm positive that the `inject` step is needed, I'm not sure if the `instantiate: false` line is necessary, or if we need to stub with an `Ember.Service`. An `Ember.Object` or even a POJO might be enough.

I don't think this matters too much though, so you can decide if it's worth the time to potentially save an import and a line or two.

Reply:
I'll leave it for now. Not a big priority relative to other things.